src/modules/site/site.service.ts

```diff
@@ -31,4 +34,23 @@ export class SiteService {
     const { URL: siteId, Sitestatus: status } = data.value[0].fields;
     return { siteId, status };
   }
+
+  async createSiteId(): Promise<string> {
+    const requestToCreateSiteId: MdmCreateNumbersRequest = this.buildRequestToCreateSiteId();
+    // TODO APIM-133: Do we want to wrap any errors in a new error?
+    const [{ maskedId: createdSiteId }] = await this.mdmService.createNumbers(requestToCreateSiteId);
+    return createdSiteId;
+  }
+
+  private buildRequestToCreateSiteId(): MdmCreateNumbersRequest {
+    // TODO APIM-133: Should we use the config for app.name for this, or are they separate concepts?
+    const applicationName = 'Estore';
+    return [
+      {
+        numberTypeId: 6,
+        createdBy: applicationName,
+        requestingSystem: applicationName,
+      },
+    ];
```

Comment:
@avaitonis do you know much about the significance of `createdBy` and `requestingSystem` here? I'm wondering if we should link these to the `app.name` config property (which is currently `estore` instead of `Estore`)

Reply:
`createdBy` and `requestingSystem` are not very important. It is fine to use app.name and have lower case `estore`, leaving `Estore` is also fine.

Reply:
I'll leave it as-is then, thank you